design/ingressroute-design.md

```diff
@@ -159,8 +159,6 @@ The following list are the options available to choose from:
 
 - **RoundRobin:** Each healthy upstream host is selected in round robin order
 - **WeightedLeastRequest:** The least request load balancer uses an O(1) algorithm which selects two random healthy hosts and picks the host which has fewer active requests. _Note: This algorithm is simple and sufficient for load testing. It should not be used where true weighted least request behavior is desired_
-- **Ring hash:** The ring/modulo hash load balancer implements consistent hashing to upstream hosts
```

Comment:
This is a bit non-related to this PR, but have a quick question:

Should the design doc be the goal of what we'd want, then the `docs` section describes what get's implemented?  Or do we only design what we'll implement?

Reply:
good point. I'll revert this.